cli/command/defaultcontextstore.go

```diff
@@ -14,6 +14,8 @@ import (
 const (
 	// DefaultContextName is the name reserved for the default context (config & env based)
 	DefaultContextName = "default"
+	// DefaultContextType is the default type for a context
+	DefaultContextType = "moby"
```

Comment:
/cc @cpuguy83 @tiborvass just to double-check; are we good with `moby` as name for the default? (or do we want it to be called different, e.g. "default" literally?)

Reply:
It's weird, because we don't even refer to moby as moby in moby.
It's also not very descriptive.

Reply:
I'd also like to have a context type for https://github.com/docker/cli/issues/2718#issuecomment-700162376

Reply:
DefaultContextType = "default" would sound weird to me, as a non descriptive name. Should be "docker engine" but considering the moby/docker naming split I'm fine with "moby"

Reply:
Could be "engine".
The problem is this encodes more than just the api implementation.

Reply:
"com.docker.engine.v1"

Reply:
Alternatively, use an empty string instead? (only show a backend if it's not the default (docker engine))?

Reply:
"moby" is what is currently returned as context type, for users who have installed Docker Desktop or Linux users having installed compose-cli binary. So here we want to just prepare replacing compose-cli current installation with CLI backend mechanism, it's probably better to not change the current behaviour ?

Reply:
On the contrary, it seems like a great time to get it right.